### PR TITLE
[Cesium] - fix wrong parameters on EllipsoidTangentPlane.fromPoints

### PR DESCRIPTION
**Pull request by zoran995**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/CesiumGS/cesium/blob/1.69/Source/Core/EllipsoidTangentPlane.js#L141-L148>>
 [source code](https://github.com/CesiumGS/cesium/blob/1.69/Source/Core/EllipsoidTangentPlane.js#L141), [documentation](https://cesium.com/docs/cesiumjs-ref-doc/EllipsoidTangentPlane.html#.fromPoints)

### CODE REVIEW

**Review by jrafidi — APPROVED** [review]
Looks good to me. Interestingly, the Cesium docs are incorrect here and don't list the first parameter as an array.

File: types/cesium/index.d.ts [hunk 1]
```diff
@@ -916,7 +916,7 @@ declare namespace Cesium {
         projectPointOntoPlane(cartesian: Cartesian3, result?: Cartesian2): Cartesian2;
         projectPointsOntoPlane(cartesians: Cartesian3[], result?: Cartesian2[]): Cartesian2[];
         projectPointsOntoEllipsoid(cartesians: Cartesian2[], result?: Cartesian3[]): Cartesian3[];
-        static fromPoints(ellipsoid: Ellipsoid, cartesians: Cartesian3): EllipsoidTangentPlane;
+        static fromPoints(cartesians: Cartesian3[], ellipsoid: Ellipsoid): EllipsoidTangentPlane;
```
Comment (jrafidi) [inline types/cesium/index.d.ts:919]: I think `ellipsoid` is optional based on the source and documentation

**Review by jrafidi — CHANGES_REQUESTED** [review]

**Comment by elibarzilay** [comment]
(@zoran995: I force-pushed this PR to trigger a new build.)

**Review by jrafidi — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
I believe it was reviewed already,
@zoran995 thx!

**Pull request merged**